en/backup-to-aws-s3-using-br.md

```diff
@@ -323,14 +333,43 @@ After creating the `Backup` CR, use the following command to check the backup st
  kubectl get bk -n test1 -o wide
  ```
 
-More `Backup` CR fields are described as follows:
+<details>
+<summary>More <code>Backup</code> CR parameter description</summary>
 
 - `.spec.metadata.namespace`: the namespace where the `Backup` CR is located.
+- `.spec.tikvGCLifeTime`: the temporary `tikv_gc_lifetime` time setting during the backup. Defaults to 72h.
+
+    Before the backup begins, if the `tikv_gc_lifetime` setting in the TiDB cluster is smaller than `spec.tikvGCLifeTime` set by the user, TiDB Operator adjusts the value of `tikv_gc_lifetime` to the value of `spec.tikvGCLifeTime`. This operation makes sure that the backup data is not garbage-collected by TiKV.
+
+    After the backup, no matter whether the backup is successful or not, as long as the previous `tikv_gc_lifetime` is smaller than `.spec.tikvGCLifeTime`, TiDB Operator will try to set `tikv_gc_lifetime` to the previous value.
+
+    In extreme cases, if TiDB Operator fails to access the database, TiDB Operator cannot automatically recover the value of `tikv_gc_lifetime` and treats the backup as failed. At this time, you can view `tikv_gc_lifetime` of the current TiDB cluster using the following statement:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    select VARIABLE_NAME, VARIABLE_VALUE from mysql.tidb where VARIABLE_NAME like "tikv_gc_life_time";
+    ```
+
+    If the value of `tikv_gc_lifetime` is set too large (usually 10m), you need to set it back to the previous value.
+
+- `.spec.cleanPolicy`: The clean policy of the backup file when the backup CR is deleted after the backup is completed.
```

Comment:
No need of `after the backup is completed.`

Reply:
Updated.